tailcfg/tailcfg.go

```diff
@@ -2253,6 +2253,9 @@ const (
 
 	// NodeAttrAutoExitNode permits the automatic exit nodes feature.
 	NodeAttrAutoExitNode NodeCapability = "auto-exit-node"
+
+	// NodeAttrSuggestExitNodeUI permits suggest exit node UI.
```

Comment:
missing a word in here?

And usual question on these: does this permit the option to enable something, or immediate cause something to happen? If the latter, I'd say "enables the suggested exit node UI".

Isn't this redundant with the server even sending suggested exit node information?

https://github.com/tailscale/corp/issues/19515 doesn't really clear it up for me. It's kinda sparse.

(Aside: the commit message is missing a `tailscale/` in the cross-repo issue reference)

Reply:
It permits being able to use the UI on clients and not tailscale cli. So I could say "enables the suggested exit node UI for clients" ?